package.json

```diff
@@ -8,11 +8,12 @@
   },
   "private": true,
   "workspaces": [
-    "packages/*/*"
+    "packages/*/*",
+    "packages/utils/parcel-watcher-watchman-js"
```

Comment:
Doesn't this already get matched by `packages/*/*`?

Reply:
Yeah, looks like it can be removed